Editor/Window/ConnectToFleetInput.cs

```diff
@@ -22,22 +28,25 @@ public class ConnectToFleetInput : StatefulInput
         private VisualElement _fleetStatus;
         private GameLiftFleetManager _fleetManager => _stateManager.FleetManager;
         private readonly StateManager _stateManager;
+        private readonly VisualElement _container;
         private Button _cancelButton;
 
-        private readonly VisualElement _container;
         private FleetStatus _fleetState;
         private List<FleetAttributes> _fleetAttributes = new List<FleetAttributes>();
 
         public ConnectToFleetInput(VisualElement container, StateManager stateManager, FleetStatus initialState)
```

Comment:
Why do we accept an initial state? As far as I can tell we only create this in one place with `ConnectToFleetInput.FleetStatus.NotCreated`

Reply:
Updated